packages/tables/frontend/sirius-components-tables/src/cells/Cell.tsx

```diff
@@ -128,9 +129,11 @@ export const Cell = memo(
         }
       }
       return (
-        <div className={classes.wrapper} onClick={() => onClick(cell)}>
-          {component}
-        </div>
+        <Tooltip title={cell.tooltipValue}>
```

Comment:
The tooltip seems a bit far from the real cell, it can almost seem to be on the next cell